### PR TITLE
Libsnowflakeclient cjwt wrapper

### PR DESCRIPTION
**Pull request by h-chaturvedi**
Adds a C Wrapper for JWT implementation. These APIs will for now be consumed by the ODBC driver.

### CODE REVIEW

**Comment by notkriswagner** [comment]
I re-ran the windows tests cause it looked like we were unable to communicate with the server.

**Comment by h-chaturvedi** [comment]
@notkriswagner There was one log line that was being dumped so many times due to a test. I have removed it now. That has been committed independent of this branch [here](https://github.com/snowflakedb/libsnowflakeclient/commit/3f7b62f31011e37d7714c69850bf7382fb74177b).

File: include/snowflake/jwtWrapper.hpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#ifndef JWT_WRAPPER_H
```
Comment (smtakeda) [inline include/snowflake/jwtWrapper.hpp]: Add copyright header

File: cpp/jwt/jwtWrapper.cpp [hunk 2]
```diff
@@ -0,0 +1,208 @@
+#include "snowflake/IJwt.hpp"
```
Comment (smtakeda) [inline cpp/jwt/jwtWrapper.cpp]: add copyright header.

File: include/snowflake/jwtWrapper.hpp (hunk 1 above)
Comment (smtakeda) [reply include/snowflake/jwtWrapper.hpp]: I think *.h is better as this is for C.

**Comment by notkriswagner** [comment]
LGTM

**Pull request merged**